### ISSUE TITLE
Invalid XML syntax.

### ISSUE THREAD

**Issue by cvdevol** [honza/textus-receptus#2]
All kinds of problems with the xml syntax.

**Comment by honza** [comment]
Can you provide some details?

**Comment by cvdevol** [comment]
As soon as I get back to my computer I'll send you some examples. The JSON files are OK though. 

Yahoo Mail: Search, Organize, Conquer 
 
  On Fri, Aug 30, 2024 at 12:23 PM, Honza ***@***.***> wrote:   


Can you provide some details?

**Comment by cvdevol** [comment]
in the file gnt.flat.xml: when loading in VisualStudio, I get 

System.Xml.XmlException: 'Name cannot begin with the ',' character, hexadecimal value 0x2C. Line 4, position 34.'

There are many more like this. I ended up converting the JSON files to xml. The resultant xml was valid. I used the followig conversion: (in Visual Basic .NET)
Dim xml = XDocument.Load(JsonReaderWriterFactory.CreateJsonReader(Encoding.UTF8.GetBytes(jsonString), New XmlDictionaryReaderQuotas()))
